core/test/fixtures/artifacts/update-trace-fixture.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright 2023 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// https://docs.google.com/spreadsheets/d/1AaYzpzWnpXQ4JB5IZzOkTO9Zf5Sm0I8dsp7MhBgthMg/edit?usp=sharing
+
+import * as puppeteer from 'puppeteer';
+
+import {LH_ROOT} from '../../../../root.js';
+import {Server} from '../../../../cli/test/fixtures/static-server.js';
+import {saveTrace, saveDevtoolsLog} from '../../../lib/asset-saver.js';
+
+/**
+ * @typedef CollectMeta
+ * @property {string} name
+ * @property {string} about
+ * @property {(page: puppeteer.Page, port: number) => Promise<LH.UserFlow>} runUserFlow
+ * @property {(artifacts: LH.Artifacts) => void} verify
+ * @property {string|false} saveTrace
+ * @property {string|false} saveDevtoolsLog
+ */
+
+/**
+ * @param {CollectMeta} collectMeta
+*/
+export async function updateTestFixture(collectMeta) {
```

Comment:
nit: this file is called `update-trace-fixture.js` but the function is called `updateTestFixture`. I think renaming this file to `update-test-fixture.js` makes sense because we get the DT log as well.